front-end/src/ui-components/InfoBox.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ReactNode, useState } from 'react';
+import styled from '@xstyled/styled-components';
+import { MdClose } from 'react-icons/md';
+
+interface Props{
+	children?: ReactNode
+	className?: string
+	content?: string
+	dismissable?: boolean
+	name: string
+	title: string
+}
+
+const InfoBox = ({ children, className, content, dismissable, name, title }: Props) => {
+	const infoBoxVisible = localStorage.getItem(name);
+	if (infoBoxVisible === null) {
+		localStorage.setItem(name, 'true');
+	}
+
+	const [infoVisible, setInfoVisible] = useState(JSON.parse(infoBoxVisible!));
```

Comment:
Do you need this JSON.parse?

Reply:
Well, I want the boolean value instead of a string

Reply:
ok then `infoBoxVisible === 'true'` should do the job and would be much easier to read.